ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -291,7 +332,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "hearingBookingDetailsGatekeeping",
-    "UserRole": "caseworker-publiclaw-gatekeeper",
```

Comment:
do we not need to give the gatekeeper access to this also?

Reply:
Yeah, don't know why that got changed. But it has been decided that it is not in the scope of the project and will be reverted the access for that feature will be given in a new story.